SoomlaiOSProfile/ProfileEventHandling.h

```diff
@@ -54,6 +56,17 @@
 #define EVENT_UP_INVITE_CANCELLED               @"up_invite_cancelled"
 #define EVENT_UP_INVITE_FAILED                  @"up_invite_failed"
 
+#define EVENT_UP_LEADERBOARDS_STARTED           @"up_leaderboards_started"
```

Comment:
@EugeneButusov where's the GET? it should be GET_LEADERBOARDS_STARTED not just LEADERBOARDS_STARTED